src/entities/trades/uniswap-v2/constants.ts

```diff
@@ -75,6 +103,13 @@ export const WBTC: Record<number, Token> = {
     'WBTC',
     'Wrapped BTC from Ethereum'
   ),
+  [ChainId.OPTIMISM_MAINNET]: new Token(
+    ChainId.OPTIMISM_MAINNET,
+    '0x68f180fcce6836688e9084f035309e29bf0a2095',
+    8,
+    'WBTC',
+    'Wrapped BTC from Ethereum'
```

Comment:
The token name on the explorer is "Wrapped BTC" https://optimistic.etherscan.io/token/0x68f180fcce6836688e9084f035309e29bf0a2095